RELEASE.md

```diff
@@ -4,18 +4,24 @@
 
 A release for the bpfman project is comprised of the following major components:
 
-- bpfman binaries
-- Core GRPC API protobuf definitions
-- Kubernetes Custom Resource Definitions (CRDs)
-- Corresponding go pkg in the form of `github.com/bpfman/bpfman` which includes the following:
+- bpfman and bpfman-api(Core GRPC API protobuf definitions) library crates
+- bpfman-cli, and bpfman-rpc(Core GRPC api shim) binary crates
```

Comment:
```suggestion
- bpfman (CLI), and bpfman-rpc (Core GRPC api shim) binary crates
```
I don't think we have anything name `bpfman-cli`. The `bpfman` binary is the CLI.

Reply:
Good point